src/base/preferences.cpp

```diff
@@ -1054,6 +1056,115 @@ void Preferences::setExecutionLogEnabled(bool b)
     setValue("Preferences/ExecutionLog/enabled", b);
 }
 
+bool Preferences::executionLogNormalType() const
```

Comment:
Isn't that a little wasteful? Maybe it's better to save one value and interpret it as a set of flags?